app/src/Location.kt

```diff
@@ -0,0 +1 @@
+data class Location(val x: Double, val y: Double)
```

Comment:
I use `Double` by default since it has more precision and it's the default literal interpretation in most languages and never think about it if the need doesn't arise.